src/shridhar_k/ProgrammingTest2_1.java

```diff
@@ -0,0 +1,31 @@
+package shridhar_k;
+
+public class ProgrammingTest2_1 {
+	char printLastConsonants(String inputStr) {
+
+		for (int i = 0; i < inputStr.length(); i++) {
+			char ch = inputStr.charAt(i);
+			if (inputStr.indexOf(ch) != inputStr.lastIndexOf(ch)) {
+				inputStr = inputStr.replace(String.valueOf(ch), "@");
+			}
+			ch = inputStr.charAt(i);
+			if ('a' == ch || 'e' == ch || 'i' == ch || 'o' == ch || 'u' == ch) {
```

Comment:
We write ch == 'a' , i know both is same, but best way is this

Reply:
Noted, Thank you